docs/docs/.vuepress/config.js

```diff
@@ -23,8 +23,27 @@ module.exports = {
     searchPlaceholder: "Search...",
     logo: "/pipedream.svg",
     repo: "PipedreamHQ/pipedream",
-
-    // Optional options for generating "Edit this page" link
+    nav: [
+      {
+        text: "v2",
+        className: "docs-version",
+        ariaLabel: "Docs Version Menu",
+        items: [
+          {
+            text: "v2",
+            link: "https://pipedream.com/docs-v2",
+            internal: true,
```

Comment:
Was this property set `true` to avoid the external link icon, or does it serve another purpose?

Reply:
This is part of that hack I had to do in order for the links to escape the parent path, it's a long story.